src/js/tech/tech.js

```diff
@@ -213,36 +160,38 @@ class Tech extends Component {
 
   /*! Time Tracking -------------------------------------------------------------- */
   manualTimeUpdatesOn() {
-    let player = this.player_;
-
     this.manualTimeUpdates = true;
 
-    this.on(player, 'play', this.trackCurrentTime);
-    this.on(player, 'pause', this.stopTrackingCurrentTime);
+    this.on('play', this.trackCurrentTime);
+    this.on('pause', this.stopTrackingCurrentTime);
     // timeupdate is also called by .currentTime whenever current time is set
 
-    // Watch for native timeupdate event
-    this.one('timeupdate', function(){
+    // Watch for native timeupdate event only
```

Comment:
We should just remove this bit now. It was originally added because at the time, not all browsers supported the timeupdate event.